CHANGELOG.md

```diff
@@ -3,7 +3,64 @@
+- plugins/logs: Add support for masking with array keys ([#6883](https://github.com/open-policy-agent/opa/issues/6883)) authored by @charlieegan3
+- tester: code nitpicks ([#7252](https://github.com/open-policy-agent/opa/pull/7252)) authored by @srenatus
+- testing: replace reflect.DeepEqual where possible ([#7286](https://github.com/open-policy-agent/opa/pull/7286)) authored by @anderseknert
+- util: Add util.Keys and util.KeysSorted ([#7285](https://github.com/open-policy-agent/opa/pull/7285)) authored by @anderseknert
+
+### Docs, Website, Ecosystem
+
+- docs: Update docker compose file in HTTP API tutorial and use addr for binding ([#7264](https://github.com/open-policy-agent/opa/issues/7264)) authored and reported by @zanliffick
+- docs: Make 'ancient' warnings closable ([#7253](https://github.com/open-policy-agent/opa/issues/7253)) authored by @srenatus reported by @konradzagozda
+- docs: Redirect opa-1 to v0-upgrade ([#7259](https://github.com/open-policy-agent/opa/pull/7259)) authored by @charlieegan3
+- docs: Use preformatted strings in fmt help ([#7263](https://github.com/open-policy-agent/opa/pull/7263)) authored by @charlieegan3
+- docs: Fix typo in k8s primer ([#7242](https://github.com/open-policy-agent/opa/pull/7242)) authored by @vicentinileonardo
+- docs: Formatting and wording fixes ([#7268](https://github.com/open-policy-agent/opa/pull/7268)) authored by @kamilturek
+- docs: Update output document of Envoy plugin. ([#7241](https://github.com/open-policy-agent/opa/pull/7241)) authored by @regeda
+
+### Miscellaneous
+
+- ci(nightly): Remove vendor w/o modproxy check ([#7292](https://github.com/open-policy-agent/opa/pull/7292)) authored by @srenatus
+- Dependency updates; notably:
+  - build(go): bump to 1.23.5 ([7279](https://github.com/open-policy-agent/opa/pull/7279)) authored by @srenatus
+  - build(deps): upgrade github.com/dgraph-io/badger to v4 (4.5.1) ([#7239](https://github.com/open-policy-agent/opa/pull/7239)) authored by @Juneezee
+  - build(deps): bump github.com/containerd/containerd from 1.7.24 to 1.7.25
+  - build(deps): bump github.com/tchap/go-patricia/v2 from 2.3.1 to 2.3.2
+  - build(deps): bump golang.org/x/net from 0.33.0 to 0.34.0
+  - build(deps): bump golang.org/x/time from 0.8.0 to 0.9.0
+  - build(deps): bump google.golang.org/grpc from 1.69.2 to 1.70.0
+  - build(deps): bump go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp from 0.58.0 to 0.59.0
+  - build(deps): bump go.opentelemetry.io/otel from 1.33.0 to 1.34.0
+  - build(deps): bump go.opentelemetry.io/otel/exporters/otlp/otlptrace from 1.33.0 to 1.34.0
+  - build(deps): bump go.opentelemetry.io/otel/exporters/otlp/otlptrace/otlptracegrpc from 1.33.0 to 1.34.0
+  - build(deps): bump go.opentelemetry.io/otel/sdk from 1.33.0 to 1.34.0
+  - build(deps): bump go.opentelemetry.io/otel/trace from 1.33.0 to 1.34.0
```

Comment:
I think they use a condense format themselves, like:
```suggestion
  - build(deps): bump go.opentelemetry.io deps to 1.34.0/0.59.0
```

Come to think of it, shouldn't we have a group defined in dependabot.yml for all the Otel stuff? 🤔

Reply:
Hm! We do, and there was only one PR, too: https://github.com/open-policy-agent/opa/pull/7283. I'm curious how these lines came to be 💭